ppa/settings.py

```diff
@@ -252,6 +252,11 @@
     },
 }
 
+
+# defaults for HathiTrust dataset rsync, per their documentation
+HATHITRUST_RSYNC_SERVER = 'datasets.hathitrust.org'
+HATHITRUST_RSYNC_PATH = ':ht_text_pd'
```

Comment:
A single colon is the normal separator between the server name and the path on the server, like you would use with scp. You use two colons when you want to specify an rsync module instead of a path — which seems to be how they have it configured.

I put the colon here in the config to make it optional; that way, we can override it and use a regular file path on our qa server in local settings for testing without having to configure an rsync module on the server.